ftgo-common/src/main/java/net/chrisrichardson/ftgo/common/DeliveryState.java

```diff
@@ -1,4 +1,4 @@
-package net.chrisrichardson.ftgo.deliveryservice.domain;
+package net.chrisrichardson.ftgo.common;
```

Comment:
It doesn't make sense for this to be in `common`.
It's belongs in a `ftgo-delivery-service-*` module.